### DISCUSSION TITLE
Integration between different web deployments

### DISCUSSION THREAD

**Discussion by hwpplayer1** [rocky-linux/rockylinux.org-hugo#62] [General]
# Integration between different web deployments

## Deploy on GitHub

Deploy on GitHub is an easy solution and it is a quick solution

## Deploy on a Dedicated Server

Deploy on a Dedicated Server may be a more comfortable solution which datacenter support engineers can change the disks easily and make a platform for Rocky up and running 24/7 with mirrors

## Deploy on shared hosting as a mirror

While physical machines are running 24/7 shared hosting services can be a stage where we fix small paragpraphs and source code.

I can not write all Cloud Computing paradigms but it sounds like Dedicated should be first as a main branch and then the others for development and fixes.

Thanks.

**Comment by FoggyMtnDrifter** [comment]
Are you suggesting changes to the way we serve the site? Currently there's no reason to switch from Vercel at this point, as has been mentioned in Slack.

**Reply by hwpplayer1** [reply]
In short No.

For a long answer I just explained how I learned from other communities. They deploy software like yours as a quick solution rather than managing server and playing with hardware.

So I started that thread for a better communication in open public

@ressonix Thanks for your understanding.
